MAINTENANCE_STANDARDS.md

```diff
@@ -0,0 +1,24 @@
+This document outlines a standardized policy for when Dependabot will cease guaranteeing support with package managers, and for when it begin supporting a new version. "Cease guaranteeing support" is intentionally distinct from "deprecating support", because just because something's not supported, doesn't necessarily mean that we need to "rip out support immediately". Instead, it means more like, "if supporting this becomes a problem, we won't fix it".
+
+These principles will need to be weighed against the unique circumstances, but without principles, we won't ever be able to make consistent decisions.
+
+Note, this builds on ideas documented internally earlier by @jeffwidman.
+
+**General support principles**
+
+We should try to align with the ideas of [SemVer](https://semver.org/), but we can’t guarantee everything will follow that.
```

Comment:
We are moving towards [SemVer2](https://github.com/dependabot/dependabot-core/pull/10434)